cmd/control-plane-machine-set-operator/main.go

```diff
@@ -146,6 +149,7 @@ func main() { //nolint:funlen,cyclop
 		Namespace:      managedNamespace,
 		OperatorName:   "control-plane-machine-set",
 		ReleaseVersion: getReleaseVersion(setupLog),
+		Recorder:       mgr.GetEventRecorderFor("controlplanemachineset-controller"),
```

Comment:
We hyphenate in other places don't we?
```suggestion
		Recorder:       mgr.GetEventRecorderFor("control-plane-machine-set-controller"),
```